Cargo.lock

```diff
@@ -5751,7 +5751,7 @@ dependencies = [
  "proc-macro2 1.0.53",
  "quote 1.0.26",
  "rustc_version 0.4.0",
- "syn 1.0.109",
+ "syn 2.0.5",
```

Comment:
nit: `2.0.15` is the latest version according to https://crates.io/crates/syn/versions.

Reply:
I also notice that.
But this is what `cargo` does.
Notice that `Cargo.toml` says just `2.0`.
Any ideas as to why this might be happening?